### ISSUE TITLE
MethodAliasGenerator doesn't generate parameter attributes

### ISSUE THREAD

**Issue by devlead** [cake-build/cake#2393]
Currently code gen omits parameter attributes, this means i.e. the `GetCallerInfo` generates wrong results when called without a context.

This source code

https://github.com/cake-build/cake/blob/730bf9b83e794583fdd3b93677c9dc0ae9fcbee8/src/Cake.Common/Diagnostics/ScriptCallerAliases.cs#L23-L28

becomes
```csharp
ScriptCallerInfo GetCallerInfo(
            string memberName = "",
            string sourceFilePath = "",
            int sourceLineNumber = 0)
```

Reference code:
 * https://github.com/cake-build/cake/blob/develop/src/Cake.Core/Scripting/CodeGen/MethodAliasGenerator.cs

**Comment by devlead** [comment]
@kcamp pefect, assigned issue to you.

**Comment by devlead** [comment]
Fixed by #2402
